collect_app/src/main/java/org/odk/collect/android/widgets/range/RangePickerWidgetUtils.kt

```diff
@@ -32,4 +33,18 @@ object RangePickerWidgetUtils {
 
         return displayedValuesForNumberPicker.toTypedArray()
     }
+
+    @JvmStatic
+    fun getProgressFromPrompt(prompt: FormEntryPrompt, listOfValues: Array<String>): Int {
+        var actualValue: BigDecimal? = null
+        val answerValue = prompt.answerValue
+        if (answerValue != null) {
```

Comment:
It would be good to add a test to drive this (and the below) `if` statement out. I think there's only one case (`null` answer value on `prompt`) that actually requires them both.